src/content/en/tools/lighthouse/audits/blocking-links.md

```diff
@@ -0,0 +1,61 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site Does Not Use Link Tags That Delay First Paint" Lighthouse audit.
+
+{# wf_updated_on: 2016-12-01 #}
+{# wf_published_on: 2016-12-01 #}
+
+# Site Does Not Use Link Tags That Delay First Paint  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Fast page loads result in higher user engagement, more pageviews, and
+improved conversion.
+
+You can improve your page load speed by inlining resources that are required
+for first paint, and deferring those that aren't.
+
+## How to pass the audit {: #how }
+
+In your report, Lighthouse lists all of the render-blocking links that it has
+detected. The goal is to reduce this number.
+
+As mentioned in [How the audit is implemented](#implementation), Lighthouse
+flags three types of render-blocking links: scripts, stylesheets, and HTML
+imports. How you optimize depends on what type of link you're working with.
+
+Note: When a resource is referred to as "critical" below, it means that the
+resource is required for first paint.
+
+* For critical scripts, consider inlining them in your HTML. For non-critical
+  scripts, consider marking them with the `async` or `defer` attributes.
+  See [Adding Interactivity with JavaScript][js] to learn more.
+* For stylesheets, consider splitting up your styles into different files,
+  organized by media query, and then adding a `media` attribute to each
+  stylesheet link. When loading a page, the browser only blocks the first
+  paint to retrieve the stylesheets that match the user's device. See
+  [Render-Blocking CSS][css] to learn more.
+* For non-critical HTML imports, mark them with the `async` attribute.
```

Comment:
For imports, `async` should be used all the time if the user can get away with it.